orca-clouddriver/src/test/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/cluster/AbstractClusterWideClouddriverTaskSpec.groovy

```diff
@@ -142,6 +143,25 @@ class AbstractClusterWideClouddriverTaskSpec extends Specification {
 
   }
 
+  def 'cluster wide task should use moniker if available or else fallback on frigga'() {
+    given:
+    def stage = new Stage<>(new Pipeline("orca"), 'clusterSelection', [
+      cluster    : cluster,
+      moniker    : moniker,
+    ])
+    when:
+    ClusterSelection selection = stage.mapTo(ClusterSelection)
+
+    then:
+    selection.getApplication() == expected
+
+    where:
+    cluster | moniker | expected
```

Comment:
I would let intellij auto-align these pipe characters to make this more readable